frappe/website/web_template/hero/hero.html

```diff
@@ -8,14 +8,14 @@ <h1>
 	</p>
 	{%- endif -%}
 	{%- if primary_action or secondary_action -%}
-	<div class="mt-4">
+	<div>
 		{%- if primary_action -%}
-		<a class="btn btn-lg btn-primary" href="{{ primary_action }}">
+		<a class="mt-4 mr-2 mr-lg-3 btn btn-lg btn-primary" href="{{ primary_action }}">
```

Comment:
Remove utility classes and style them like this:

```css
.hero-content .btn-primary {
	//
}
```

Bootstrap's utility classes make it harder to override them.